content/en/tracing/advanced/connect_logs_and_traces.md

```diff
@@ -119,6 +119,29 @@ This enables automatic trace ID injection for `winston`, `bunyan`, and `pino`.
 
 **Note**: Automatic injection only works for logs formatted as JSON.
 
+{{% /tab %}}
+{{% tab ".NET" %}}
+
+The .NET Tracer uses the [LibLog][2] library to automatically inject trace IDs into your application logs. It contains transparent built-in support for injecting into [NLog][3], [Log4Net][4], and [Serilog][5].
+
+Enable injection in the .NET Tracer’s [configuration][1] by setting `DD_LOGS_INJECTION=true` through environment variables or the configuration files. Additionally, injection can be enabled in the code:
```

Comment:
Let's add a line break between these two thoughts and we're good to go.

Reply:
done!